src/zarr/testing/strategies.py

```diff
@@ -60,7 +60,7 @@ def v2_dtypes() -> st.SearchStrategy[np.dtype]:
 )
 array_names = node_names
 attrs = st.none() | st.dictionaries(_attr_keys, _attr_values)
-keys = st.lists(node_names, min_size=1).map(lambda x: "/".join(x))
+keys = st.lists(node_names, min_size=1).map("/".join)
```

Comment:
Cool! 🤩

Did not even realize this was valid in Python. Other languages certainly have this concept. Will keep this in mind for the future